src/antelope/chains/EVMChainSettings.ts

```diff
@@ -383,6 +411,14 @@ export default abstract class EVMChainSettings implements ChainSettings {
 
         this.processNftContractsCalldata(response.contracts);
         const shapedNftData = this.shapeNftRawData(shapedIndexerNftData, response.contracts);
+
+        // if the owner does not comes from the indexer, let's set it
+        shapedNftData.forEach((nft) => {
```

Comment:
this should only be set for erc721, which is why line 409 has a ternary like 
`owner: isErc1155 ? undefined : nftResponse.owner,`
so here we should also only set owner if it's 1155

Reply:
I updated to use `owner: nftResponse.owner ?? account`